src/fights.ts

```diff
@@ -1335,6 +1335,14 @@ const freeRunFightSources = [
       ) {
         set("_gingerbreadCityTurns", 1 + get("_gingerbreadCityTurns"));
       }
+    },
+    {
+      requirements: () => [
+        new Requirement([], {
+          // eslint-disable-next-line libram/verify-constants
+          forceEquip: $items`carnivorous potted plant`.filter((item) => have(item)),
```

Comment:
It looks like the plugin just upgraded; you may not need this

Reply:
Sadly not, upgraded around 11 hours ago and KoLMafia only added the data around 2 hours ago.